docs/getting-started.mdx

```diff
@@ -0,0 +1,144 @@
+After installing the [Nitric CLI](/docs/installation), you can start building a project.
+
+### Using the `new` command
+
+The Nitric CLI's `new` command provides prompts to scaffold new projects from templates. In this example we'll select the TypeScript starter template:
+
+```bash
+nitric new
+? What is the name of the stack? hello-world
+? Choose a template: official/TypeScript - Starter
+```
+
+Navigate to the new project directory and install the NPM dependencies:
+
+```bash
+cd hello-world
+
+npm install
+```
+
+Your project should now look like this:
+
+```txt
++--functions/
+|  +-- hello.ts
++--node_modules/
+|  ...
++--package-lock.json
++--package.json
++--README.md
+```
+
+### Running your app
+
+The Nitric CLI can run your application locally for development and testing. The run command needs to know which source files represent your application code. In this example all of our app code is inside the `functions/` folder, so we provide a glob pattern to match that folder.
+
+```bash
+nitric run ./functions/*.ts
+```
+
+Output:
+
+```txt
+SUCCESS  Created Dev Image! (4s)
+⠻ Waiting for Local Services to be ready (3s)Registered Gateway Plugin
+Starting Child Process
+Services listening on: 0.0.0.0:50051
+Waiting for active workers
+Starting Worker Supervisor
+Starting Gateway, 0 workers currently available
+running membrane 🚀
+SUCCESS  Started Local Services! (4s)
++-----------------+------------------------------------+
+| RUNDIR          | /Users/you/.nitric/run/hello-world |
+| GATEWAYADDRESS  | :9001                              |
+| MEMBRANEADDRESS | localhost:50051                    |
+| MINIOENDPOINT   | localhost:9000                     |
++-----------------+------------------------------------+
+SUCCESS  Started Functions! (0s)
+Local running, use ctrl-C to stop
+
+Api  | Endpoint
+main | http://localhost:9001/apis/main
+```
+
+Take a look in the `hello.ts` file, you'll see it declares an API named `main` with a single route `GET /hello/:name`.
+
+When running locally Nitric hosts all your APIs under the path:
+
+```txt
+http://localhost:9001/apis/[api_name]/
+```
+
+in this case the URL for the API is:
+
+```txt
+http://localhost:9001/apis/main/
+```
+
+You can test the API using cURL, your browser, or any other HTTP client:
+
+```bash
+curl http://localhost:9001/apis/hello/hello/John
```

Comment:
is two hellos correct?

Reply:
Woops, hangover - that's what the current template has, but we're renaming the API in the stack template to 'main'. I've fixed this in the doc, I'll go update the template.